synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -167,8 +167,8 @@ void CanvasResize::on_dialog_shown()
 	if (is_image_checked) set_image_flags(toggle_ratio_wh->get_active());
 
 	// Always enable width/height link
-	set_canvas_flags(true);
-	refresh_wh_toggle_widgets();
+//	set_canvas_flags(true);
+//	refresh_wh_toggle_widgets();
```

Comment:
This was the only way to default turn on width/height link

But the side effect was the code disregards user choice and always toggles the link back on when the dialog is shown. Code imitated from Canvas Properties class.